### DISCUSSION TITLE
[Simulation question]: Handling oxygen transportation

### DISCUSSION THREAD

**Discussion by sdevkaushik** [topas-nbio/TOPAS-nBio#34] [Simulation-question]
### Simulation question

Hi, I use neutrons as a primary source of radiation in my simulations. Neutrons further generates several secondary charged particle species including oxygen, carbon, alpha, electrons, protons etc. I was wandering which particular physics model I should include to accurately simulate oxygen transportation in my simulations. Currently, I am using a combination of G4EmDNAPhysics_option2 and G4EmDNAPhysics_option4 which I beleive has a common model for all HZE particles: GenericIon_G4DNAIonisation.  

### TOPAS-nBio category

Geometry

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by tmasilela** [comment]
Hi,

Apologies that you've had to wait some time for a response. Unfortunately there are no Geant4-DNA physics processes and models for track structure simulations of neutrons (see the [Geant4-DNA webpage](http://geant4-dna.in2p3.fr/styled-3/styled-8/index.html)). What you could do is first launch a condensed history simulation with [TOPAS](https://github.com/OpenTOPAS/OpenTOPAS/tree/main), collect a [phase space](https://opentopas.readthedocs.io/en/latest/parameters/scoring/phasespace.html) at your volume of interest, then use that phase space in your track structure simulation.

Some other points about your post:

- You should only use one electromagnetic physics option, I'm assuming that if you supply two in your TOPAS parameter file and no error is given then probably only the first one is being used. I'm not too sure of this though.
- If you decide to go through with the above suggestion of first doing a condensed history simulation you could use something like [QGSP_BERT_HP_EMZ](https://opentopas.readthedocs.io/en/latest/parameters/physics/reference.html). "HP" activates the high precision neutron model for precise tracking of neutrons below ~20 MeV if I remember correctly, and "EMZ" in this case is electromagnetic option 4. I'm not sure if this is the "best" physics list for neutron simulations but it is likely reasonable.

Hope that helps.

All the best,
Thongchai

**Reply by sdevkaushik** [reply]
Hi Thongchai,

Thanks for replying.

We follow the same approach of using first condensed history simulations to record secondary species spectra and their relative dose contributions at any initial neutron energy. Then, we use this information in the track structure for DNA-level simulations at our lab. Thanks for your suggestions though!
I also developed some additional models for accurate thermal neutron transport simulations in the condensed history approach that I would like to share at some point. 
I just wanted to know if there are any additional models to improve oxygen, nitrogen, and carbon ion simulations in TOPAS-nBio.

Thanks,
Sachin

**Reply by tmasilela** [reply]
To the best of my knowledge, no, there are no additional models that can be activated in TOPAS-nBio.

As you pointed out in your original post, all ions are simulated using the G4DNAIonisation process class. There might be some small differences depending on which [option](https://doi.org/10.1002/mp.13048) you choose, but - unless I'm mistaken - I don't think you'll see an "improvement" in the transportation of those ion species regardless of the option you choose.
